easybuild/easyconfigs/c/CGAL/CGAL-4.11-foss-2017b.eb

```diff
@@ -0,0 +1,40 @@
+name = 'CGAL'
+version = '4.11'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'strict': True}
+
+source_urls = ['https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-%(version)s']
+sources = [SOURCE_TAR_XZ]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Boost', '1.65.1'),
+    ('MPFR', '3.1.6'),
+    ('GMP', '6.1.2'),
+    ('libGLU', '9.0.0'),
```

Comment:
Hmm, `Python` is not included as a dependency anymore, while it was before (see also `versionsuffix`)?